molecule/src/molecule_ionize.cpp

```diff
@@ -444,433 +419,423 @@ void MoleculePkaModel::_loadSimplePkaModel()
+        {"6400020420000|64000204210006400020421000|64000204200008200022201000820002222000082000222010008200022220000", 5.10f, 0.00f},
+        {"6400020420000|64000204210006400020421000|64000204100006400020420000820002220100082000222010008200022220000", 10.68f, 0.00f},
+        {"8200022210000|6400020420000|64000204300008200022210000", 12.81f, 0.90f},
+        {"6400020411000|64000204110006400020411000|640002041100064000204110006400020411000", 14.90f, 0.00f},
+        {"7300021310000|6400020421000|640002042100073000213100008200022201000", 13.00f, 0.00f},
+        {"7300021320000|64000204210006400020421000|64000204210007300021320000820002220100073000213300008200022201000", 7.89f, 0.00f},
+        {"16200022210000|6400020420000|162000222100006400020421000", 9.43f, 0.00f},
+        {"7300021320000|64000204210006400020421000|64000204200007300021320000820002220100073000213300008200022201000", 7.20f, 0.00f},
+        {"7300021320000|64000204210006400020421000|64000204110007300021320000820002220100073000213300008200022201000", 6.70f, 0.00f},
+        {"7300021320000|64000204210006400020421000|64000204400007300021320000820002220100073000213200008200022201000", 7.72f, 0.29f},
+        {"7300021320000|166000206220006400020421000|6400020421000730002132000082000222010008200022201000162000222200007300021311000", 7.20f, 0.00f},
+        {"8200022210000|15500020531000|8200022201000820002221000082000222100008200022220000", 6.30f, 0.10f},
+        {"7300021320000|64000204210006400020421000|64000204400007300021320000820002220100073000213300008200022201000", 8.45f, 0.00f},
+        {"7300021320000|166000206220006400020421000|640002042100073000213200008200022201000820002220100064000204110007300021311000", 6.70f, 0.00f},
+        {"7300021320000|166000206220006400020421000|640002044000073000213200008200022201000820002220100064000204110006400020421000", 3.55f, 1.05f},
+        {"7300021320000|166000206220006400020421000|640002042100073000213200008200022201000820002220100064000204210008200022220000", 5.00f, 0.00f},
+        {"7300021320000|166000206220006400020421000|640002042100073000213200008200022201000820002220100073000213110007300021311000", 7.40f, 0.00f},
+        {"82-10023110000|6400020421000|640002043000082-100231100008200022201000", 3.40f, 0.00f},
+        {"7300021320000|166000206220006400020421000|640002042100073000213200008200022201000820002220100064000204210008200022201000", 4.57f, 0.00f},
+        {"7300021320000|64000204210007300021330000|73000213200007300021320000820002220100064000204200006400020420000", 3.60f, 0.00f},
+        {"7300021320000|64000204210006400020421000|73000213110007300021320000820002220100064000204210008200022201000", 9.69f, 0.00f},
+        {"82-10023110000|6400020421000|640002042100082-100231100008200022201000", 3.20f, 0.00f}
+    };
 
-    for (auto i = 0; i < NELEM(advanced_pka_model); i++)
+    void LoadPkaDefToModel(RedBlackStringObjMap<Array<float>>& adv_model, PkaDef* from, PkaDef* const end)
```

Comment:
It's better to have iterators of the same type - both `const` in this case; not to "save on" intermediate variables. :)